<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UiEncoding</class>
 <widget class="QWidget" name="UiEncoding">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Encoding/Decoding Tool</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="encoding_TabName_General">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="encoding_TabName_General_2">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_EncodingAscii">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>ASCII</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="textEdit_EncodeDecode_General_Ascii">
           <property name="acceptDrops">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_EncodingHex">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>HEX</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="textEdit_EncodeDecode_General_HEX">
           <property name="acceptDrops">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="2">
        <layout class="QVBoxLayout" name="verticalLayout_EncodingDec">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>DEC</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="textEdit_EncodeDecode_General_DEC">
           <property name="acceptDrops">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_EncodingBIN">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>BIN</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="textEdit_EncodeDecode_General_BIN">
           <property name="acceptDrops">
            <bool>false</bool>
           </property>
           <property name="html">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2'; font-size:8.25pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_EncodeBase64">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Base64</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="textEdit_EncodeDecode_General_Base64">
           <property name="acceptDrops">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="2">
        <layout class="QFormLayout" name="formLayout_EncodingHashes">
         <property name="topMargin">
          <number>20</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>MD5</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="encoding_lineEdit_MD5">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>SHA512</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="encoding_lineEdit_SHA512">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_13">
           <property name="text">
            <string>SHA3 512</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="encoding_lineEdit_SHA3_512">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_17">
           <property name="text">
            <string>KECCAK 512</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="encoding_lineEdit_KECCAK512">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_EncodingAlphanum">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Alphanumeric string</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="textEdit_EncodeDecode_General_AlphanumericStrings">
           <property name="acceptDrops">
            <bool>false</bool>
           </property>
           <property name="html">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2'; font-size:8.25pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="1">
        <layout class="QVBoxLayout" name="verticalLayout_Number">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Number</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="textEdit_EncodeDecode_General_Number">
           <property name="acceptDrops">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="2">
        <layout class="QVBoxLayout" name="verticalLayout_EncodingRaw">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>RAW Data</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="AdvancedPlainTextEdit" name="textEdit_EncodeDecode_General_RawData"/>
         </item>
        </layout>
       </item>
       <item row="3" column="0" colspan="3">
        <layout class="QHBoxLayout" name="horizontalLayout_Footer">
         <item>
          <widget class="QPushButton" name="pushButton_SaveAs">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Save as</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_EncodeDecode_General_LoadBinary">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Load binary file</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_ElapsedTime">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_EncodeDecode_General_ClearAll">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Clear all</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_11">
      <attribute name="title">
       <string>Web</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_17">
       <item row="0" column="0">
        <widget class="QLabel" name="label_39">
         <property name="text">
          <string>URL Decoded</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_40">
         <property name="text">
          <string>URL Encoded</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QTextEdit" name="textEdit_EncodeDecode_UrlEncoded"/>
       </item>
       <item row="1" column="1">
        <widget class="QTextEdit" name="textEdit_EncodeDecode_UrlDecoded"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_42">
         <property name="text">
          <string>HTML Decoded</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_41">
         <property name="text">
          <string>HTML Encoded</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QTextEdit" name="textEdit_EncodeDecode_HtmlDecoded"/>
       </item>
       <item row="3" column="1">
        <widget class="QTextEdit" name="textEdit_EncodeDecode_HtmlEncoded"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AdvancedPlainTextEdit</class>
   <extends>QPlainTextEdit</extends>
   <header>base/extensions/AdvancedPlainTextEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
